trifinger_simulation/pinocchio_utils.py

```diff
@@ -27,42 +27,59 @@ def __init__(
             for link_name in tip_link_names
         ]
 
-    def forward_kinematics(self, joint_positions) -> typing.List[np.ndarray]:
-        """Compute end-effector positions for the given joint configuration.
+    def forward_kinematics(
+        self,
+        joint_positions: typing.List[np.ndarray],
+        joint_velocities: typing.Optional[np.ndarray] = None,
+    ) -> typing.Union[
+        typing.List[np.ndarray],
+        typing.Tuple[typing.List[np.ndarray], typing.List[np.ndarray]],
+    ]:
+        """Compute end-effector positions (and velocities) for the given joint configuration.
 
         Args:
             joint_positions:  Flat list of angular joint positions.
+            joint_positions (optional):  Flat list of angular joint
```

Comment:
1. should be `joint_velocities` here
2. I would rather move the "(optional)" on the other side of the colon (i.e. "joint_velocities: Optional.  Flat list...").